pandas/_libs/parsers.pyx

```diff
@@ -591,6 +579,9 @@ cdef class TextReader:
             self.parser.quotechar = ord(quote_char)
 
     cdef _make_skiprow_set(self):
+        cdef:
+            int64_t i
```

Comment:
For anything that is used to index Python objects you should opt for `Py_ssize_t`

Reply:
I see, my reason to have this as ```int64_t``` is because below we have the line:

```cython
            for i in self.skiprows:
                parser_add_skiprow(self.parser, i)
```

Which calls ```parser_add_skiprow```, ```parser_add_skiprow``` is defined here:

https://github.com/pandas-dev/pandas/blob/7673357191709036faad361cbb5f31a802703249/pandas/_libs/parsers.pyx#L207

and ```i``` is in the place of ```row```, I tried to match the type, to avoid (potential) casting.

Reply:
I think that should also be a Py_ssize_t; I don't know what effort it would require to update that but OK to split off on a separate PR too